### PR TITLE
Add Bank Hapoalim

### PR DESCRIPTION
**Pull request by dleshem**
Alexa rank: [6,413](https://www.alexa.com/siteinfo/bankhapoalim.co.il)

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -98,6 +98,13 @@ websites:
       - hardware
     doc: https://www.bca.co.id/en/Individu/Produk/E-Banking/Klik-BCA
 
+  - name: Bank Hapoalim
+    url: https://www.bankhapoalim.co.il/
+    img: poalim.png
+    twitter: PoaliMarkeTweet
+    facebook: BankHapoalimSocial
```
Comment (Carlgo11) [inline _data/banking.yml]: Can't find any page with that handle.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 1 above)
Comment (dleshem) [reply _data/banking.yml]: Twitter: https://twitter.com/PoaliMarkeTweet
Facebook: https://www.facebook.com/BankHapoalimSocial

Both work for me.

Comment (Carlgo11) [reply _data/banking.yml]: The thing is, the Facebook link doesn’t work for me or the script. image

Comment (dleshem) [reply _data/banking.yml]: Perhaps they limited the page to Israeli users...

**Comment by kmpoppe** [comment]
There's **Facebook Global Pages** which allows brands to have different sites for countries under the same handle. Additionally one can set country restrictions to each and every page. 
And even though their Twitter page is way more active than craigslist's we've got the same problem, what good is listing these accounts, when only a very limited amount of people can communicate with them.

**Comment by dleshem** [comment]
This is the biggest bank in Israel :) The Facebook page is viewable in Israel and has 110k followers, which is 2-3% of Israel's adult population.

**Comment by dleshem** [comment]
@Carlgo11 Bottom line: the Facebook page exists, is popular, active and accessible to Israeli users (this site's target audience). Do you want me to remove it so the tests pass, or disable/fix the test?

**Comment by dleshem** [comment]
Bump

**Comment by Carlgo11** [comment]
Remove anything that breaks the build.

**Comment by dleshem** [comment]
> Remove anything that breaks the build.

Done

**Review by Carlgo11 — APPROVED** [review]

**Review by Jawshy — APPROVED** [review]

**Pull request merged**